released/packages/coq-mathcomp-multinomials/coq-mathcomp-multinomials.1.4/opam

```diff
@@ -15,7 +15,7 @@ install: [
 ]
 depends: [
   "coq" {(>= "8.7" | < "8.11~")}
-  "coq-mathcomp-ssreflect" {(>= "1.8.0" & < "1.11~") | (= "dev")}
+  "coq-mathcomp-ssreflect" {(>= "1.8.0" & < "1.10~")}
```

Comment:
Are there non-technical reasons for excluding ssreflect 1.10 for version 1.4? Because it works fine, as far as I can tell (I've used it locally for a while).

Reply:
Sometime, I get lost with all these versions. Let's try it.

Reply:
Ah, coq-mathcomp-finmap is incompatible with math-comp 1.10 (or t least, declared to be so). So anyway, opam is going to force select math-comp 1.9. Am I wrong?

Reply:
finmap-1.3.4 works fine with math-comp 1.10 (and this is reflected in the OPAM archive since this weekend). Here is what my current `opam list` looks like:
```
coq-mathcomp-finmap       1.3.4         pinned to version 1.3.4
coq-mathcomp-multinomials 1.4           A Multivariate polynomial Library for the Mathematical Components Library
coq-mathcomp-ssreflect    1.10.0        pinned to version 1.10.0
```